tsconfig.json

```diff
@@ -21,7 +21,6 @@
 		"strictNullChecks": true,
 		"target": "es2015",
 		"paths": {
-			"@clayui/*": ["node_modules/@clayui/*/src"],
```

Comment:
I think the problem with this is that we need the storybook to work correctly and also the inference in the IDE for development. I think we can define a new "paths" just for the declaration configuration file so this only changes in the types' build output, what do you think?

Reply:
Yeah that was a concern of mine as well. I figured I would let you investigate it further though since you might have a bit more context.

My main goal was making sure that the published `./lib` folder contained no references or dependencies to `./src`. Because right now in clay's published packages, it references src which I don't think was originally intended. See line 79 of [this file](https://unpkg.com/browse/@clayui/core@3.104.0/lib/vertical-nav/VerticalNav.d.ts). 

Making this change fixed that, but I think we could make this change _just_ in the definitions tsconfig

Reply:
I see @bryceosterhaus, I'll delve deeper into this thanks for the work on this.